legacy-projects.md

```
@@ -587,29 +591,50 @@ Projects kept out of the main [README](README.md) list — which only features a
 
-#### Integration
+#### Integration Guide
 
-- [flashg1/SolarCharger](https://github.com/flashg1/SolarCharger) - Home Assistant custom integration that optimizes electric vehicle charging based on available surplus solar energy (Python · ⭐ 16).
-- [JustChr/HAgoe_steve](https://github.com/JustChr/HAgoe_steve) - Home Assistant integration providing smart-charging orchestration for go-e wallboxes by coordinating with SteVe (OCPP) backend and optimizing based on solar production, electricity prices, and home battery state (Python · ⭐ 0).
+- [Veton-ev/Veton-EMS-Integration](https://github.com/Veton-ev/Veton-EMS-Integration) - A developer guide for integrating Veton/Phoenix Contact CHARX EV chargers into Energy Management Systems via Modbus, MQTT, REST, and OCPP with runnable code examples (⭐ 0).
 
 #### Libraries
 
-##### Makefile
+##### C#
```

markdownlint-cli2 (0.23.1)

[warning] No space inside hashes on closed atx style heading

(MD020, no-missing-space-closed-atx)